src/NavList/NavList.tsx

```diff
@@ -15,6 +15,14 @@ import {defaultSxProp} from '../utils/defaultSxProp'
 import {useId} from '../hooks/useId'
 import useIsomorphicLayoutEffect from '../utils/useIsomorphicLayoutEffect'
 
+const getSubnavStyles = (depth: number) => {
+  return {
+    paddingLeft: depth > 0 ? depth + 2 : 0, // Indent sub-items
```

Comment:
Setting `paddingLeft` to `0` overrides the default padding from `ActionList.Item` and causes root-level items have no left padding:

![Arrow pointing at a nav list item without left padding](https://github.com/primer/react/assets/4608155/58d4a9b8-2a9f-4f28-979b-4be20f3b57fd)


```suggestion
    paddingLeft: depth > 0 ? depth + 2 : null, // Indent sub-items
```

Reply:
Good catch. Thanks!